modules/oeFax/controller/FaxServer.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * Fax Server SMS Module Member
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2019 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+$ignoreAuth = 1;
+require_once(__DIR__ . "/../../../interface/globals.php");
+
+use OpenEMR\Common\Crypto\CryptoGen;
+
+class FaxServer
+{
+    private $baseDir;
+    private $crypto;
+    private $authToken;
+
+    public function __construct()
+    {
+        $this->baseDir = $GLOBALS['OE_SITE_DIR'] . DIRECTORY_SEPARATOR . "messageStore";
+        $this->cacheDir = $GLOBALS['OE_SITE_DIR'] . '/documents/logs_and_misc/_cache';
+        $this->serverUrl = (isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] === 'on' ? "https" : "http") . "://" . $_SERVER['HTTP_HOST'];
```

Comment:
PLACEMARKER for onoging review

Reply:
I going to bring this back to 5.0.2 when done.

Reply:
dang, this is a lot of code :)
need some shut eye; will continue review tomorrow :)